CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+
+## Assumptions
+
+1. **You're familiar with [GitHub](https://github.com) and the [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)(PR) workflow.**
+2. **You've read the MeiliSearch [documentation](https://docs.meilisearch.com) and the [README](/README.md).**
+3. **You know about the [MeiliSearch community](https://docs.meilisearch.com/learn/what_is_meilisearch/contact.html). Please use this for help.**
+
+## How to Contribute
+
+1. Make sure that the contribution you want to make is explained or detailed in a GitHub issue! Find an [existing issue](https://github.com/meilisearch/meilisearch-rails/issues/) or [open a new one](https://github.com/meilisearch/meilisearch-rails/issues/new).
+2. Once done, [fork the meilisearch-rails repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) in your own GitHub account. Ask a maintainer if you want your issue to be checked before making a PR.
+3. [Create a new Git branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository).
+4. Review the [Development Workflow](#workflow) section that describes the steps to maintain the repository.
+5. Make the changes on your branch.
+6. [Submit the branch as a PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) pointing to the `main` branch of the main meilisearch-rails repository. A maintainer should comment and/or review your Pull Request within a few days. Although depending on the circumstances, it may take longer.<br>
+ We do not enforce a naming convention for the PRs, but **please use something descriptive of your changes**, having in mind that the title of your PR will be automatically added to the next [release changelog](https://github.com/meilisearch/meilisearch-rails/releases/).
+
+## Development Workflow
+
+### Setup <!-- omit in toc -->
+
+```bash
+bundle install
+```
+
+### Tests and Linter <!-- omit in toc -->
+
+Set the credentials of the MeiliSearch instance as environment variables.
+
+```
+MEILISEARCH_HOST="http://127.0.0.1:7700"
+MEILISEARCH_API_KEY="masterKey"
+```
+
+Each PR should pass the tests and the linter to be accepted.
+
+```bash
+# Tests
+docker pull getmeili/meilisearch:latest # Fetch the latest version of MeiliSearch image from Docker Hub
+docker run -p 7700:7700 getmeili/meilisearch:latest ./meilisearch --master-key=masterKey --no-analytics=true
+bundle exec rspec
+# Launch a single test in a specific file
+bundle exec rspec spec/integration_spec.rb -e 'should include _formatted object'
+# Launch tests in a specific folder or file 
+bundle exec rspec spec/integration_spec.rb
+# Linter
+bundle exec rubocop lib/ spec/
+# Linter with fixing
+bundle exec rubocop -a lib/ spec/
+```
+
+### Playground <!-- omit in toc -->
+
```

Comment:
```suggestion

First, you need to run a MeiliSearch instance:

```bash
docker run -p 7700:7700 getmeili/meilisearch:latest ./meilisearch --no-analytics=true
```bash
```

Reply:
Sorry I cannot add a real suggestion because of the ```